base/src/org/spin/queue/util/QueueManager.java

```diff
@@ -179,7 +182,15 @@ public QueueManager withEntity(PO entity) {
 		}
 		return this;
 	}
-	
+
+	public QueueManager withEntity(int tableId, int recordId) {
+		if (getContext() == null || getTransactionName() == null) {
+			throw new AdempiereException("@not.found@");
```

Comment:
please change by AdempiereException("@NotFound@");

Reply:
ok, I will change this value